src/ucp/api/ucp.h

```diff
@@ -1089,8 +1089,10 @@ void ucp_worker_release_address(ucp_worker_h worker, ucp_address_t *address);
  * communication progress.
  *
  * @param [in]  worker    Worker to progress.
+ *
+ * @return Non-zero if any communication was progressed, zero otherwise.
```

Comment:
in the current implementation this is a counter, but i don't see a reason to make it mandatory in the future as well